packages/cactus-plugin-persistence-ethereum/README.md

```diff
@@ -0,0 +1,237 @@
+# `@hyperledger/cactus-plugin-persistence-ethereum`
+
+This plugin allows `Cactus` to persist Ethereum data into some storage (currently to a `PostgreSQL` database, but this concept can be extended further).
+Data in the database can later be analyzed and viewed in a GUI tool.
```

Comment:
@outSH If someone comes in and submits a complicated pull request with a giant refactor that makes it so that this plugin supports SQLite as well, would you rather `a)` accept that with the additional complexity or `b)` prefer to ask that contributor to compartmentalize their change into a different plugin altogether?

The reason I ask is because if in that case `b)` is preferred then I'd name this plugin `cactus-plugin-persistence-ethereum-pg` so that in the imaginary scenario the other person can call their implementation `cactus-plugin-persistence-ethereum-sqlite` without any sort of confusion about which plugin does what.

This is just a question, not a change request (unless you'd prefer `b)` in which case then I'll ask to rename the plugin to `cactus-plugin-persistence-ethereum-pg`.

Reply:
@petermetz I want to support / be able to support other data storages in the future, and for this reason data access layer is separated from the monitoring functions of the main plugin, so I'd say option `a)`. Ideally such PR would introduce another db-client and some sort of dynamic selector that would choose data storage based on input args to the plugin.

Reply:
@outSH Kk!